bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/LabelImpl.java

```diff
@@ -78,6 +76,6 @@ public Boolean isVisible() {
      */
     @Override
     public String getValue() {
-        return ComponentUtils.translate(properties.get(PN_TITLE, this.defaultTitle), PN_TITLE, properties, i18n);
+        return ComponentUtils.translate(properties.get(PN_TITLE, "\n"), PN_TITLE, properties, i18n);
```

Comment:
If there is no title in JCR, why are you returning an empty line ?

Reply:
@rismehta In order to make it consistent with AF v1 and as per the suggestions of @ujjwal5 we have returned an empty line instead of empty string. I can even change it to empty string too if required. Pls suggest.